testdata/picard/sam/summary_alignment_stats_test_supplementary.sam

```diff
@@ -0,0 +1,8 @@
+@HD	VN:1.0	SO:coordinate
+@SQ	SN:chr6	LN:101
+@SQ	SN:chr8	LN:202
+@RG	ID:0	SM:Hi,Momma!	LB:whatever	PU:me	PL:ILLUMINA
+SL-XAV:1:1:0:700#0/2	137	chr6	1	255	101M	*	0	0	NAATTGTTCTNAGTTTCTCGGTTTATGTGCTCTTCCAGGTGGGTAACACAATAATGGCCTTCCAGATCGTAAGAGCGACGTGTGTTGCACNAGTGTCGATC	&0::887::::6/646::838388811/679:87640&./2+/-4/28:3,536/4''&&.78/(/554/./02*)*',-(57()&.6(6:(0601'/(,*	RG:Z:0
```

Comment:
it would be good if this example were more realistic, so that the primary read were only aligned for, say 60 bases and the supplementary read would be aligned for the remaining 41 bases. also, the bases and the qualities should be the same for the read and its supplemental alignment....

Reply:
@PolinaBevad are you going to add a more realistic test?